doctest/extensions/doctest_mpi.h

```diff
@@ -1,166 +1,188 @@
-#pragma once
+#ifndef DOCTEST_MPI_H
+#define DOCTEST_MPI_H
+
+#ifdef DOCTEST_CONFIG_IMPLEMENT_WITH_MPI_MAIN
+#define DOCTEST_CONFIG_IMPLEMENT
+#endif // DOCTEST_CONFIG_IMPLEMENT_WITH_MPI_MAIN
 
 #ifdef DOCTEST_CONFIG_IMPLEMENT
 
-#include "doctest/extensions/mpi_sub_comm.h"
+#include "mpi_sub_comm.h"
 #include "mpi_reporter.h"
 #include <unordered_map>
+#include <exception>
```

Comment:
where is this used?

Reply:
It is included in origin file when `DOCTEST_CONFIG_IMPLEMENT` isn't defined, and I haven't removed it to be prudent.

I checked again and it doesn't use, so I delete it.